recipes/conversation/conversation.js

```diff
@@ -24,7 +24,7 @@ var credentials = config.credentials;
 var WORKSPACEID = config.conversationWorkspaceId;
 
 // these are the hardware capabilities that TJ needs for this recipe
-var hardware = ['microphone', 'speaker'];
+var hardware = ['microphone', 'speaker']; //full capabilities list ['led', 'servo', 'microphone', 'speaker','camera']
```

Comment:
ignore previous (deleted) comment -- just noticed this is in the conversation recipe.  there doesn't seem to be a point to enumerate the full capabilities list in this recipe, it only needs 2 capabilities.

my previous suggestion was to put something like this in tjbotlib instead:

```
var allHardware = ['led', 'servo', 'microphone', 'speaker','camera'];
```